sdk/attestation/attestation/review/attestation.api.md

```diff
@@ -51,12 +28,16 @@ export interface AttestationCertificateManagementBody {
 // @public
 export class AttestationClient {
     constructor(credentials: TokenCredential, instanceUrl: string, options?: AttestationClientOptions);
-    // (undocumented)
-    attestation: Attestation;
+    // Warning: (ae-forgotten-export) The symbol "AttestationResult" needs to be exported by the entry point index.d.ts
+    attestOpenEnclave(report: Uint8Array, options?: AttestOpenEnclaveOptions): Promise<AttestationResponse<AttestationResult_2>>;
```

Comment:
Let's have `AttestationResult` exported from `src/index.ts` and make sure an other type with the same name is not.

Reply:
Ah, found the source of the duplicate....